IssueTracker/Models.Response/UserToken.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Models.Info
+{
+    public class UserToken
+    {
+        public UserToken()
+        {
+            Token = string.Empty;
+        }
+    
+        public string Token { get; set; }
```

Comment:
change to: public string Token { get; set; } = string.Empty;

Reply:
I changed